### ISSUE TITLE
Fix for python 3.11 warning

### ISSUE THREAD

**Issue by jderam** [jderam/hyperborea3#147]
When running tests under python 3.11
```
hyperborea3/hyperborea3/db.py:17: DeprecationWarning: path is deprecated. Use files() instead. Refer to https://importlib-resources.readthedocs.io/en/latest/using.html#migrating-from-legacy for migration advice.
    with path("hyperborea3", "hyperborea.sqlite3") as db_path:
```
Implement update to address the warning

**Comment by jderam** [comment]
https://importlib-resources.readthedocs.io/en/latest/using.html#migrating-from-legacy

I think it's appropriate to address this when/if dropping support for python 3.8, since the new `importlib.resources.files` doesn't become available until python 3.9+

**Comment by jderam** [comment]
This has been addressed in 0.6.0
